openstack-mixin/panels.libsonnet

```diff
@@ -838,29 +884,35 @@ local utils = commonlib.utils;
       + timeSeries.standardOptions.withDecimals(0),
 
     volumeUsage:
-      commonlib.panels.generic.timeSeries.percentage.new(
-        'Volume usage',
+      commonlib.panels.generic.timeSeries.base.new(
+        'Volume usage in GB',
         targets=[t.volumeUsage],
-        description='The percent of volume storage in use for Cinder.',
+        description='Volume storage usage in bytes in use for Cinder.',
       )
-      + timeSeries.standardOptions.withUnit('percentunit')
+      + timeSeries.standardOptions.withUnit('decgbytes')
       + timeSeries.options.legend.withPlacement('right')
-      + timeSeries.standardOptions.withMax(1)
-      + timeSeries.standardOptions.withMin(0),
+      + timeSeries.options.legend.withCalcs('lastNotNull')
+      + timeSeries.options.legend.withDisplayMode('table')
+      + timeSeries.options.legend.withSortBy('Last *')
+      + timeSeries.options.legend.withSortDesc(true)
+      + timeSeries.fieldConfig.defaults.custom.withLineWidth('1'),
 
     backupUsage:
-      commonlib.panels.generic.timeSeries.percentage.new(
-        'Backup usage',
+      commonlib.panels.generic.timeSeries.base.new(
+        'Backup usage in GB',
         targets=[t.backupUsage],
-        description='The percent of backup storage in use for Cinder.',
+        description='Backup storage usage in bytes in use for Cinder.',
```

Comment:
Including units in titles/descriptions is generally discouraged. Also, this is not a 'usage' dashboard anymore by standard definition. I would change the title to `Backup storage used` or something similar.